plugins/wazuh-core/server/services/data-path.ts

```diff
@@ -18,13 +19,48 @@ export interface IDataPathService {
   createDirectory(subDirectory?: string): void;
   createDataDirectoryIfNotExists(directory?: string): void;
   getDataDirectoryRelative(directory?: string): string;
+  setup(): Promise<void>;
+  start(): Promise<void>;
+  stop(): Promise<void>;
 }
 
 export class DataPathService implements IDataPathService {
   private dataPath: string;
 
-  constructor(globalConfig: DataPathConfig) {
-    this.dataPath = globalConfig.path.data || './data';
+  constructor(private logger: Logger, globalConfig: DataPathConfig) {
+    this.dataPath = globalConfig.path.data;
+  }
+
+  /**
+   * Setup the service
+   */
+  async setup(): Promise<void> {
+    this.logger.debug('DataPathService: Setup');
```

Comment:
The logger already include a name defined here, so you can remove the "tag" or rename the logger: https://github.com/wazuh/wazuh-dashboard-plugins/pull/7586/files/ad0f577b9030a41aa8a7815bd2e877797b32369a..cf4be5ba6ede367f9cb12e30be8c2f805e29b820#diff-4a1db4ef18224f4e63a401f98799477682f7cb9b81577a1c8b51404d765b0115R59

```suggestion
    this.logger.debug('Setup');
```